lti/views/link_selection.py

```diff
@@ -0,0 +1,16 @@
+from pyramid.view import view_config
+from lti.util.lti_launch import lti_launch
+
+@view_config(route_name='link_selection', renderer='lti:templates/module_item_configurations/new_module_item_configuration.html.jinja2', request_method='POST')
+@lti_launch
+def link_selection(request):
+  """
+    Renders the form that teachers see to configure the module item.
+    This view is only used for lms's that support link selection
+  """
+  return {
+    'launch_presentation_return_url': request.params['launch_presentation_return_url'],
```

Comment:
Is it possible to get into a state where launch_presentation_return_url is not in the parms? Should this be wrapped in a try/catch?

Reply:
Where this is an lti launch we can feel pretty safe that the value will be there because it is part of the lti standard. We can't really support any LMS that doesn't implement at the bare minimum a basic functional lti launch.